src/verifiers/issuerIdentity/builder.ts

```diff
@@ -89,7 +66,12 @@ export const issuerIdentityVerifierBuilder = (verifiers: IssuerIdentityVerifierD
         const documentData = getData(document);
         const verificationResultsDeferred = documentData.issuers.map((issuer, issuerIndex) => {
           const identityProofType = issuer.identityProof?.type;
-          if (!identityProofType) throw new Error("");
+          if (!identityProofType)
```

Comment:
We dont check for this scenario (identityProofType) in v3.

Reply:
v3 does not allow for undef types, v2 has that for legacy reasons (OpenCerts)